sources/packages/backend/libs/sims-db/src/entities/sfas-application.model.ts

```diff
@@ -129,4 +130,257 @@ export class SFASApplication extends BaseSFASApplicationModel {
     nullable: false,
   })
   extractedAt: Date;
+
```

Comment:
As a recommendation, please follow the pattern for the file. We can follow the same if no blank lines were added between existing properties or add blank lines to existing properties.

Reply:
I will follow the pattern of the existing file.